src/modules/ethereum/transactions/getTransactionReceipt.ts

```diff
@@ -0,0 +1,28 @@
+import type { TransactionReceipt } from 'ethers';
+import { SLOW_MODE } from 'src/env/config';
+import type { Chain } from 'src/modules/networks/Chain';
+import type { Networks } from 'src/modules/networks/Networks';
+import { sendRpcRequest } from 'src/shared/custom-rpc/rpc-request';
+import { wait } from 'src/shared/wait';
+
+export async function getTransactionReceipt({
```

Comment:
`TransactionPoller` is already performing this work
I think we should be able to tap into it from the UI

Reply:
I was thinking about this, but didn't get the easy solution. Will think again